src/site/content/en/blog/compress-images-avif/index.md

```diff
@@ -0,0 +1,348 @@
+
+* [Chrome](https://www.google.com/chrome/) version 85 or later
+* [cmake](https://cmake.org/)
+* [git](https://git-scm.com/)
+* [ninja](https://ninja-build.org/)
+
+### Building command line encoder avifenc
+
+#### 1. Get the code
+
+```shell
+git clone https://github.com/AOMediaCodec/libavif.git
+```
+
+#### 2. Change directory to libavif
+
+```shell
+cd libavif
+```
+
+There are many different ways you can configure avifenc and libavif to build.
+You can find more information at [libavif](https://github.com/AOMediaCodec/libavif).
+We are going to build avifenc so that it is statically linked to the
+AV1 encoder and decoder library, [libaom](https://aomedia.googlesource.com/aom).
+
+#### 3. Get and build libaom
+
+Change to the libavif extensions directory.
+
+```shell
+cd ext
+```
+
+The next command will pull the libaom source code
+and build libaom statically.
+
+```shell
+./aom.cmd
+```
+
+Change directory to libavif.
+
+```shell
+cd ..
+```
+
+#### 4. Build the command line encoding tool, avifenc
+
+It is a good idea to create a build directory for avifenc.
+
+```shell
+mkdir build
+```
+
+Change to the build directory.
+
+```shell
+cd build
+```
+
+Create the build files for avifenc.
+
+```shell
+cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=0 -DAVIF_CODEC_AOM=1 -DAVIF_LOCAL_AOM=1 -DAVIF_BUILD_APPS=1 ..
+```
+
+Build avifenc.
+
+```shell
+make
+```
+
+You have successfully built avifenc!
+
+### Understanding the avifenc command line parameters
+
+avifenc uses the command-line structure:
+
+```shell
+./avifenc [options] input.file output.avif
+```
+
+The basic parameters for avifenc used in this tutorial are:
+
+<table class="w-table">
+  <thead>
+    <tr>
+      <th>avifenc</th>
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>--min 0</td><td>Set min quantizer for color to 0</td>
+    </tr>
+    <tr>
+      <td>max 63</td><td>Set max quantizer for color to 63</td>
+    </tr>
+    <tr>
+      <td>--minalpha 0</td><td>Set min quantizer for alpha to 0</td>
+    </tr>
+    <tr>
+      <td>--maxalpha 63</td><td>Set max quantizer for alpha to 63</td>
+    </tr>
+    <tr>
+      <td>-a end-usage=q</td><td>Set the rate control mode to Constant Quality (Q) mode</td>
+    </tr>
+    <tr>
+      <td>-a cq-level=Q</td><td>Set quantize level for both color and alpha to Q</td>
+    </tr>
+    <tr>
+      <td>-a color:cq-level=Q</td><td>Set quantize level for color to Q</td>
+    </tr>
+    <tr>
+      <td>-a alpha:cq-level=Q</td><td>Set quantize level for alpha to Q</td>
+    </tr>
+    <tr>
+      <td>-a tune=ssim</td><td>Tune for SSIM (default is to tune for PSNR)</td>
+    </tr>
+    <tr>
+      <td>--jobs J</td><td>Use J worker threads (default: 1)</td>
+    </tr>
+    <tr>
+      <td>--speed S</td><td>Set encoder speed from 0-10 (slowest-fastest. default: 6)</td>
+    </tr>
+  </tbody>
+</table>
+
+The cq-level option sets the quantize level (0.63) to control the quality for color or alpha.
```

Comment:
Please change "0.63" to either "0..63" (two dots) or "0-63". This is intended to show the range of the quantize level, which is from 0 to 63, inclusive. Since we use "0-10" for the range of encoder speed at line 276, perhaps "0-63" is better?